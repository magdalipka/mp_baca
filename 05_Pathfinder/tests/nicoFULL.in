10 10
1 0 1 0 0 1 0 1 0 0
0 0 1 0 1 0 0 0 1 0
0 1 0 0 1 0 1 0 0 0
0 0 1 0 0 0 1 0 1 1
1 0 0 0 1 0 1 0 1 0
0 0 1 1 0 0 1 0 1 0
0 1 0 0 0 1 1 0 0 0
1 1 0 1 1 1 0 0 1 0
0 0 0 1 0 0 0 1 1 0
0 1 1 1 1 1 1 0 0 0
10
r 7 0 8 9
r 9 5 0 0
r 0 3 0 0
r 1 9 8 9
r 4 9 4 1
i 7 0 8 9
i 9 5 0 0
i 0 3 0 0
i 1 9 8 9
i 4 9 4 1

