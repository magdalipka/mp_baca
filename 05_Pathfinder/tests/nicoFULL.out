r E E N N N W W N N N N E E N N W
r S S W W N N N N N W W S S S S W S W W S S W W S
r N E N E E N E E S S W S W W S S W W S
r S W S S E S E E N E E N N E E S E E N N W
r W S S S E E N N E E S S S S S S W S W W
i E E N N N W W N N N N E E N N W
i S S W W N N N N N W W S S S S W S W W S S W W S
i N E N E E N E E S S W S W W S S W W S
i S W S S E S E E N E E N N E E S E E N N W
i W S S S E E N N E E S S S S S S W S W W
